components/surveyCards.tsx

```diff
@@ -0,0 +1,17 @@
+import { type FC } from 'react'
+import { surveyCardInterface } from '../pages/types'
+
```

Comment:
Add the interface for the props in this page rather than creating a new file.
The `pages` directory is for file routing so there will be a page `/types` with the way you currently have it set up so you can just move the interface here and delete that other file.